src/msonedrive/Module.md

```diff
@@ -0,0 +1,17 @@
+Connects to Microsoft OneDrive and provides information on the files which have been stored on OneDrive.
+
+# Module Overview
+This module contains operations for accessing the items stored in OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | 1.2.0   |
+| Microsoftgraph REST API | v1.0          |
+
+## Supported Operations
+- Get item from root directory
+- Get item from a non-root directory
+
+## Sample Application
+Please refer the sample application on the README.
```

Comment:
```suggestion
Refer the sample application on the README.
```
Can we link to the README file?

Reply:
Based on the comment from Danesh, I will add some code snippets of supported API calls here. Hence I will remove the highlighted sentence.